spring-integration-core/src/main/java/org/springframework/integration/channel/management/PollableChannelManagement.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.integration.channel.management;
+
+import org.springframework.jmx.export.annotation.ManagedMetric;
+import org.springframework.jmx.support.MetricType;
+
+/**
+ * @author Gary Russell
+ * @since 4.2
+ *
+ */
+public interface PollableChannelManagement extends MessageChannelMetrics {
+
+	@ManagedMetric(metricType = MetricType.COUNTER, displayName = "MessageChannel Receive Count")
+	int getReceiveCount();
+
+	@ManagedMetric(metricType = MetricType.COUNTER, displayName = "MessageChannel Receive Count")
+	long getReceiveCountLong();
```

Comment:
Unfortunately, we didn't deprecate the int methods; also, for "normal" volume users, we shouldn't force them to use longs everywhere (in their client). The longs were added for very high volume apps.